discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
```

Comment:
i think he means external as in not inline

the docs are still inline, they aren't manually written somewhere else